applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/usteer.js

```diff
@@ -91,25 +91,55 @@ function collectWlanAPInfoEntries(connectioninfo_table_entries, wlanAPInfos) {
 	}
 };
 
+function tootltip(mac, IP, hostname) {
+	var body= E([]);
+	var i=0;
+	body.appendChild(E('div', mac));
+	if (typeof IP !== 'undefined') {
+		while ((typeof IP['ipaddrs'][i] !== 'undefined') && (String(IP['ipaddrs'][i]).length > 0)) {
+			body.appendChild(E('div', IP['ipaddrs'][i]));
+			i++;
+		}
+		
+		i=0;
+		while ((typeof IP['ip6addrs'][i] !== 'undefined') && (String(IP['ip6addrs'][i]).length > 0)) {
+			body.appendChild(E('div', IP['ip6addrs'][i]));
+			i++;
+		}
+	}
+	if (hostname !== '') {
+		body.appendChild(E('div', hostname));
+	}
+	return body;
+}
 
 function collectWlanAPInfos(compactconnectioninfo_table_entries, wlanAPInfos) {
 	for (var wlan in wlanAPInfos) {
-		var hostl = '';
+		var hostl = E([]);
 		for (var mac in Clients) {
 			if (typeof Clients[mac] !== 'undefined')
 				if (typeof Clients[mac][wlan] !== 'undefined')
-					if (String(Clients[mac][wlan]['connected']).valueOf() == String('true').valueOf()) {
+					if (String(Clients[mac][wlan]['connected']).valueOf() === String('true').valueOf()) {
```

Comment:
Must one coerce everything to strings here? Doesn't the following work identically?:

`Clients[mac][wlan]['connected'] === 'true'`

Reply:
Somehow that does not work. But `if (String(Clients[mac][wlan]['connected']).valueOf() === 'true') ` that does work. So I will leave it like this.